README.md

```diff
@@ -1,32 +1,58 @@
 <center><img src='resources/icons/logoLarge.svg' height='100px' style='border-radius: 10px;'/></center>
 
-> LiveBot is specifically designed for bots; it cannot be used for user accounts.
+---
+
+**NOTICE**
+
+LiveBot is specifically designed for bots; it cannot be used for user accounts. The LiveBot developers take no resposibility for the actions executed through LiveBot.
+
+---
 
 An app that allows you to be inside a bot!
 
 At the moment, LiveBot is still in alpha. Installation instructions can be found below.
 
-## Discord
+## Support Server
 If you would like to get the latest information on the project, talk with the developers, post suggestions, get help, etc., join the official LiveBot [Discord server!](https://discord.gg/NG4rgqSgzx)
 
-## What is this?
+## Preview
 <img src='https://i.imgur.com/m0HvVmn.png' alt='LiveBot image preview' height='300px' style='border-radius: 10px;'/>
 
-LiveBot is a program built with electron and discord.js, that will allow you to have discord controlling a bot. You are able to view messages in channels, send messages, interact with people, and see what is going on in the servers your bot is in. LiveBot can only be used on bot accounts, if you find a way to use it on user accounts we're in no way liable for anything that might happen.
+LiveBot is a program built with Electron and Discord.js that will allow you to control your bot from a client resembling the official one for users. You are able to view messages in channels, send messages, interact with people and see what is going on in the servers your bot is in.
+
+It had been built so that the style and flow is similar to discord, and while it is not exact, it is pretty close. It is also built from scratch using no frameworks, just straight HTML, CSS, and JavaScript. Due to this, and having a very small development team, additional features may take a while to be added. The point is to built from scratch, we know very well that it's not the fastest way to make something like this. However, LiveBot is slowly being filled with cool exclusive features like the pull-up user settings menu, the soon to come embed builder, native support for scripts, and more!

```

Comment:
Would prefer this started with "LiveBot has". This modification has been made by me in the temporary PR_Test branch